typings/index.ts

```diff
@@ -1,45 +1,76 @@
-/**
- *  Mask options
- *  @param {string} type - Type of the mask to be applied
- *  @param {string} redactWith - Character or word to be used as the redacted part
- *  @param {number} percentage - Percentage of the value to apply the mask on
- *  @param {boolean} complementary - Whether the complemetary part of the percentage should be masked
- */
-export type Mask = PercentageMask | UndefineMask | ReplaceMask;
+export type Mask =
+  | PercentageMask
+  | CenterPercentageMask
+  | UndefineMask
+  | ReplaceMask;
 
 export interface PercentageMask {
   type: "percentage";
-  redactWith?: "*" | "•" | "[redacted]" | string;
+  /**
+   * Character to be used as the redacted part
+   * @default "•"
+   */
+  redactWith?: "*" | "•" | string;
+  /**
+   * Percentage of the value to apply the mask on
+   * @default 100
+   */
   percentage?: number;
+  /**
+   * Which part of the value to redact
+   * @default "left"
+   */
+  position?: "left" | "right";
+}
+
+export interface CenterPercentageMask {
```

Comment:
Can we extend the Percentage mask here and overwrite the position type?

Reply:
Unfortunately no... When we use `extends`, all types must match, and `center` does not match `left | right`